examples/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Do we recommend using `venv` anywhere? if so we should ad `.venv` or whichever name we recommend

Reply:
i don't think anything uses that